### PR TITLE
Chronos/Nano: Remove redundant APIs and apply new APIs in Chronos

### PR DESCRIPTION
**Pull request by TheaperDeng**
- [x] Forecaster uts
- [x] TSPipeline uts
- [x] Nano uts

### CODE REVIEW

File: python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py [hunk 1]
```diff
@@ -245,8 +225,7 @@ def quantize(self, model,  # remove the type requirement for type checking
                  accuracy_criterion: dict = {'relative': 0.99, 'higher_is_better': True},
                  timeout=0,
                  max_trials=1,
-                 input_sample=None,
-                 return_pl=False
+                 input_sample=None
                  ):
```
Comment (jason-dai) [inline python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py:229]: I think you should move the parameters that matters to the the user most to the front of the list, such as
```python
    def quantize(self, model,  # remove the type requirement for type checking
                 precision='int8',
                 accelerator=None,
                 calib_dataloader: DataLoader = None,
                 val_dataloader: DataLoader = None,
                 metric: Optional[Metric] = None,
                 accuracy_criterion: dict = {'relative': 0.99, 'higher_is_better': True},
                 method='fx',
                 backend='inc',
                 conf: Optional[str] = None,
                 approach='static',
                 tuning_strategy='bayesian',
                 timeout=0,
                 max_trials=1,
                 input_sample=None
                 ):
```

Comment (TheaperDeng) [reply python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py:229]: sure, I will reorganize the order

**Comment by TheaperDeng** [comment]
http://10.112.231.51:18888/job/BigDL-NanoTests-PR-Validation/96/

**Comment by TheaperDeng** [comment]
@zhentaocc please also have a review, especially the nano part

**Comment by zhentaocc** [comment]
LGTM

**Pull request merged**